src/pkgcheck/checks/repo.py

```diff
@@ -1,10 +1,10 @@
 import os
 
+from charset_normalizer import is_binary
```

Comment:
The problem with the tests is that the selected file has content of `..\n`, which is decided as `is_binary = True`, which isn't correct.